src/ui/map.js

```diff
@@ -451,6 +451,15 @@ class Map extends Camera {
             throw new Error(`maxPitch must be less than or equal to ${defaultMaxPitch}`);
         }
 
+        // iOS 15.4 introduced a rendering bug with antialias set to `true`. Disables antialias for these devices.
+        if (!!options.antialias && window.navigator.userAgent && /\b(iPad|iPhone|iPod)\b/.test(window.navigator.userAgent)) {
+            const iosVersion = window.navigator.userAgent.match(/OS ((\d+_?){2,3})\s/);
+            if (iosVersion && iosVersion[1] === '15_4') {
```

Comment:
Have we confirmed that this doesn't happen on 15.3? Would it be worth it to also check for 15.5 or other future versions in case this isn't in the next Safari minor version?

Reply:
I think it's worth adding 15.5 as well. Worst case we have slightly worse maps for a couple of months. Best case we avoid broken maps for months.